locale/zh-tw/security.md

```diff
@@ -19,7 +19,7 @@ Node.js 官方為了安全研究員們及負責的公開揭露而展開了一個
 
 ## 回報第三方模組中的臭蟲
 
-第三方模組中的安全性臭蟲應向其維護者回報，且應透過 [HackerOne](https://hackerone.com/nodejs-ecosystem) 來協調。關於此流程的細節可參考安全性工作小組的 [repository](https://github.com/nodejs/security-wg/blob/master/processes/third_party_vuln_process.md)。
+第三方模組中的安全性臭蟲應向其維護者回報，且應透過 [HackerOne](https://hackerone.com/nodejs-ecosystem) 來協調。關於此流程的細節可參考安全性工作小組的[檔案庫](https://github.com/nodejs/security-wg/blob/master/processes/third_party_vuln_process.md)。
```

Comment:
I left "repository" untranslated intentionally. Since this term has no translation on GitHub, I think it may confuse our readers.
But I will leave the decision to other reviewers :)

Reply:
Vote for keeping term "repository"

Reply:
Oh, the new Git translation has translated repository to "版本庫", do you guys want to apply that?

Reply:
It depends on if readers can connect "版本庫" with "repository". "repository" used here refers to GitHub repository, as I said above, there's no translation of it on GitHub.

We can either keep it untranslated or translate it to `版本庫（repository）` or something like that. I prefer the former anyway.